src/app/units/states/portfolios/portfolios.scss

```diff
@@ -10,3 +10,17 @@
     }
   }
 }
+
+.grade-icon {
+  color: #fff;
+  font-size: 1em;
+  border-radius: 100%;
+  width: 2.25em;
+  height: 2.25em;
+  font-weight: 100;
+  margin: 0 auto;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  background-color: #333333;
+}
```

Comment:
This looks duplicated from the grade styling above

Reply:
This is for the 'All' button that was using the styling from old grade-icon scss. Since migrating, it doesn't have access to that styling which is why I duplicated it. Is there a better way to do this? 
<img width="268" alt="image" src="https://user-images.githubusercontent.com/49971210/167034481-b8f256ba-4a04-49ec-a3f3-35bc1b75103c.png">

Changing the 'All' button to a grade-icon component won't work given the way it's built, as it will be considered an invalid grade and will show an F instead.